AsyncDisplayKit/Layout/ASLayoutableExtensibility.h

```diff
@@ -0,0 +1,25 @@
+//
+//  ASLayoutableExtensibility.h
+//  AsyncDisplayKit
+//
+//  Created by Michael Schneider on 3/29/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
```

Comment:
This may not be relevant, but if you can locate the header that edge insets is defined in, let's try using only that header to more specifically document the dependencies